ccloud/beginner-cloud/cleanup.sh

```diff
@@ -27,13 +27,20 @@ CLUSTER_NAME="demo-kafka-cluster"
 CLUSTER=$(ccloud kafka cluster list | grep $CLUSTER_NAME | tr -d '\*' | awk '{print $1;}')
 #echo "CLUSTER: $CLUSTER"
 CLIENT_CONFIG="/tmp/client.config"
+CONNECTOR_NAME="datagen_ccloud_pageviews"
+CONNECTOR=$(ccloud connector list | grep $CONNECTOR_NAME | tr -d '\*' | awk '{print $1;}')
 
 ##################################################
 # Cleanup
-# - Delete the Kafka topics, Kafka cluster, environment, and the log files
+# - Delete the managed Kafka connector, Kafka topics, Kafka cluster, environment, and the log files
 ##################################################
 
-echo -e "\n# Cleanup: delete topics, kafka cluster, environment"
+echo -e "\n# Cleanup: delete connector, topics, kafka cluster, environment"
+if [[ ! -z "$CONNECTOR" ]]; then
+    echo "ccloud connector delete $CONNECTOR"
+    ccloud connector delete $CONNECTOR 1>/dev/null
+fi
+
```

Comment:
@awalther28 should the following line also be optimized to remove `connect-configs connect-offsets connect-status`?

Reply:
Correct, I have removed the unnecessary topic deletions.